src/B_read_from_files.py

```diff
@@ -443,7 +443,7 @@ def check_output_directory(settings, input_excel_file):
     os.mkdir(output_folder + INPUTS_FOLDER)
 
     path_from = os.path.abspath(input_excel_file)
-    path_to = os.path.abspath(output_folder + INPUT_TEMPLATE_EXCEL_XLSX)
+    path_to = os.path.abspath(os.path.join(output_folder + INPUT_TEMPLATE_EXCEL_XLSX))
```

Comment:
```suggestion
    path_to = os.path.abspath(os.path.join(output_folder, INPUT_TEMPLATE_EXCEL_XLSX))
```
os.path.join replaces you adding strings and slashes. So, you should have:
`os.path.join(folder_name, file_name). You need to replace the `+` with `,` and remove the slash from `INPUT_TEMPLATE_EXCEL_XLSX` and the like.